pkg/boot/universalpayload/universalpayload_test.go

```diff
@@ -79,14 +79,14 @@ address sizes	: 39 bits physical, 48 bits virtual
 				},
 			},
 			wantErr:  nil,
-			wantAddr: 0x200000 + trampolineOffset,
+			wantAddr: 0x200000 + 0xa000,
 			wantMemSegments: []kexec.Range{
-				{Start: 0x200000, Size: 0},                       // Device Tree Info
-				{Start: 0x200000 + rsdpTableOffset, Size: 0},     // ACPI Data
-				{Start: 0x200000 + tmpHobOffset, Size: 0},        // HOBs for bootloader
-				{Start: 0x200000 + tmpStackOffset, Size: 0},      // boot env (tmp stack)
-				{Start: 0x200000 + trampolineOffset, Size: 0},    // boot env (trampoline)
-				{Start: 0x200000 + uplImageOffset, Size: 0x5100}, // FIT image
+				{Start: 0x200000, Size: 0x5100},     // Device Tree Info
+				{Start: 0x200000 + 0x6000, Size: 0}, // ACPI Data
+				{Start: 0x200000 + 0x7000, Size: 0}, // HOBs for bootloader
+				{Start: 0x200000 + 0x8000, Size: 0}, // HOBs for bootloader
+				{Start: 0x200000 + 0x9000, Size: 0}, // boot env (tmp stack)
+				{Start: 0x200000 + 0xa000, Size: 0}, // boot env (trampoline)
```

Comment:
Could you please explain why you changed the definition of the memory segments from the previously used variables to constants?

Reply:
Previously, we used hard code offset for each component in UPL payload loading process. Now we will calculate the offset at run time. Let me add more comments in file.